build.sh

```diff
@@ -31,6 +33,7 @@ fi
 TAG=$AWS_ACCOUNT_ID.dkr.ecr.$AWS_REGION.amazonaws.com/community-app:$CIRCLE_SHA1
 docker build -t $TAG \
   --build-arg NODE_ENV=$NODE_ENV \
+  --build-ard SERVER_API_KEY=$SERVER_API_KEY
```

Comment:
Trailing `\` is missing

Reply:
My bad :see_no_evil: 

Not sure how I missed it